crates/fuel-core/src/schema/block.rs

```diff
@@ -89,7 +89,7 @@ impl Block {
     }
 
     async fn consensus(&self, ctx: &Context<'_>) -> async_graphql::Result<Consensus> {
-        let query = BlockQueryContext(ctx.data_unchecked());
+        let query: &Box<dyn QueryData> = ctx.data_unchecked();
```

Comment:
```suggestion
        let query: &Database = ctx.data_unchecked();
```

And the same in other places=)